core/model/src/testFixtures/kotlin/com/google/samples/apps/nowinandroid/core/model/data/AuthorFixtures.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid.core.model.data
+
+import kotlin.random.Random
+
+fun Random.nextFakeAuthor(
```

Comment:
Why are you using random values to create Test Fixtures? Wouldn't this make the tests prone to flakiness, as a test could fail with one version of the random value, but succeed with another (and thus you can't replicate test failures)